iree/tools/iree_translate_lib.h

```diff
@@ -13,9 +13,9 @@ namespace iree_compiler {
 int runIreeTranslateMain(int argc, char **argv);
 
 // NOTE: We are transitioning from the main compiler being based on
-// the MLIR translation library (i.e. iree-translate) to a dedicated tool
-// called ireec. When this is done, the above should go away and this file
-// should be renamed to ireec_lib.h.
+// the MLIR translation library (i.e. iree-compile) to a dedicated tool
+// called iree-compile. When this is done, the above should go away and this
+// file should be renamed to iree-compile_lib.h.
```

Comment:
"we are moving from iree-compile to iree-compile" 🤔 
(fine to address this note in a future change, but let's keep the comment accurate)

Reply:
Too many rounds of renames...